packages.dhall

```diff
@@ -1,12 +1,13 @@
 let mkPackage =
-      https://raw.githubusercontent.com/purescript/package-sets/psc-0.12.5-20190427/src/mkPackage.dhall sha256:0b197efa1d397ace6eb46b243ff2d73a3da5638d8d0ac8473e8e4a8fc528cf57
+      https://raw.githubusercontent.com/purescript/package-sets/psc-0.13.2-20190725/src/mkPackage.dhall sha256:0b197efa1d397ace6eb46b243ff2d73a3da5638d8d0ac8473e8e4a8fc528cf57
 
 let upstream =
-      https://raw.githubusercontent.com/purescript/package-sets/psc-0.12.5-20190427/src/packages.dhall sha256:6b17811247e1f825034fa4dacc4b8ec5eddd0e832e0e1579c2ba3b9b2a1c63fe
+      https://raw.githubusercontent.com/purescript/package-sets/psc-0.13.2-20190725/src/packages.dhall sha256:60cc03d2c3a99a0e5eeebb16a22aac219fa76fe6a1686e8c2bd7a11872527ea3
+
 
 let overrides =
       { halogen =
-          upstream.halogen // { version = "v5.0.0-rc.4" }
+          upstream.halogen // { version = "v5.0.0-rc.5" }
       , halogen-vdom =
           upstream.halogen-vdom // { version = "v6.1.0" }
       , remotedata =
```

Comment:
I believe `remotedata` at this point is in the regular set and doesn't need to be overriden

Reply:
Cool, I'll check if I can remove it